killbill-integration-tests/core/test_refund_credit.rb

```diff
@@ -308,5 +308,41 @@ def test_refund_and_cancel_with_item_adjustments
       assert_equal(refreshed_account.account_cba, 166.67)
     end
 
+    def test_find_credit_by_id
+
+      @child_account = create_child_account(@account)
+
+      # Create new credit
+      credit = create_account_credit(@child_account.account_id, 12.0, 'USD', 'Child credit', @user, @options)
+
+      # Verify if the returned list has now one element
+      get_credit = credit.find_by_id(credit.credit_id ,@options)
```

Comment:
I'm confused: I thought `credit.credit_id` didn't exist (https://github.com/killbill/killbill/issues/899)?

Reply:
@pierre  It's doesn't exist yet. I just want to leave the test ready.  Or you prefer to wait until [Issue 899](https://github.com/killbill/killbill/issues/899) has solved?

Reply:
Yes, let's wait (feel free to keep the code but comment it out with a link to the bug).

Fixing https://github.com/killbill/killbill/issues/899 means fixing other clients as well (e.g. Java models) so it might take a bit of time until we do it.

Reply:
Ok @pierre.